pkg/networkservice/chains/nsmgr/unix_test.go

```diff
@@ -39,6 +39,7 @@ import (
 )
 
 func Test_Local_NoURLUsecase(t *testing.T) {
+	t.Skip("https://github.com/networkservicemesh/sdk/issues/1118")
```

Comment:
Note: This is disabled because with p2mp rework forwarder should recv fd from the nsmgr. Currently it is not working in these unit tests by https://github.com/networkservicemesh/sdk/issues/1118 but these scenarios work well in integration tests.